skimage/filters/thresholding.py

```diff
@@ -1,10 +1,15 @@
+import math
```

Comment:
``math`` is faster than ``numpy`` for scalars, it's therefore good practice to use it.